site/app/views/grading/ElectronicGraderView.php

```diff
@@ -210,6 +210,111 @@ public function statusPage(
         ]);
     }
 
+    public function statPage($users) {
+
+		$return = <<<HTML
+        
+		<div class="content_upload_content">
+
+HTML;
+        $this->core->getOutput()->addBreadcrumb("Bulk Upload Forensics", $this->core->buildUrl(array('component' => 'submission', 'page' => 'statPage')));
```

Comment:
When clicked it brings the user to a page that says gradeable item doesn't exist (seems like the action just needs to be changed).